flask/src/test/groovy/net/corda/gradle/flask/FlaskPluginTest.groovy

```diff
@@ -167,4 +174,32 @@ class FlaskPluginTest {
         }
         assertEquals("net.corda.gradle.flask.test.AlternativeMain", prop.mainClassName)
     }
+
+    @Test
+    @DisplayName("Check that shutdown hooks are executed on flask jars")
+    void shutdownHookTest() {
```

Comment:
Should this test have its own `@org.junit.jupiter.api.Timeout`, just in case something goes wrong?